jobs/jobs.go

```diff
@@ -66,14 +63,12 @@ type getLatestVersionData struct {
 	installedData         data.InstalledData
 	clusterID             data.ClusterID
 	availableComponentVsn data.AvailableComponentVersion
-	sgc                   data.KubeSecretGetterCreator
+	k8sResources     k8s.ResourceInterfaceNamespaced
```

Comment:
looks like this needs `go fmt`ing - the type should vertically align with the others in the struct